src/services/BackgroundSchedulerService/BackgroundSchedulerService.ts

```diff
@@ -35,18 +36,35 @@ const registerPeriodicTask = async (task: PeriodicTask, exposureNotificationServ
         stopOnTerminate: false,
       },
       async taskId => {
-        log.debug({
-          category: 'background',
-          message: `runPeriodicTask: ${taskId}`,
-        });
-        try {
-          await task();
-        } catch (error) {
-          log.error({
-            category: 'background',
-            message: 'runPeriodicTask',
-            error,
-          });
+        // All background tasks come through this same callback
+        switch (taskId) {
+          case 'app.covidshield.poll-notifications': {
+            try {
+              await PollNotifications.checkForNotifications();
+            } catch (error) {
+              log.error({
+                category: 'background',
+                message: `runPollNotificationsTaskError`,
+                error,
+              });
+            }
+            break;
+          }
+          default: {
+            log.debug({
+              category: 'background',
+              message: `runPeriodicTask: ${taskId}`,
+            });
+            try {
+              await task();
+            } catch (error) {
+              log.error({
+                category: 'background',
+                message: 'runPeriodicTask',
+                error,
+              });
+            }
+          }
```

Comment:
had to refactor slightly when merging latest, this is currently untested, but *should* still work